README.md

```diff
@@ -158,15 +161,26 @@ spec:
       name: Push to Remote Azure Repo
       action: azure:repo:push
       input:
-        branch: "main"
+        branch: <MY_AZURE_REPOSITORY_BRANCH>
         sourcePath: ./sub-directory
         gitCommitMessage: Add ${{ parameters.name }} project files
 
+    - id: pullRequestAzureRepo
+      name: Create a Pull Request to Azure Repo
+      action: azure:repo:pr
+      input:
+        sourceBranch: <MY_AZURE_REPOSITORY_BRANCH>
+        targetBranch: "main"
+        repoId: <MY_AZURE_REPOSITORY>
+        title: ${{ parameters.name }}
+        project: <MY_AZURE_PROJECT>
+        supportsIterations: false
+
     - id: register
       name: Register
       action: catalog:register
       input:
-        repoContentsUrl: "dev.azure.com?owner=<MY_AZURE_PROJECT>&repo=<MY_AZURE_REPOSITORY>&organization=<MY_AZURE_ORGANIZATION>"
+        repoContentsUrl: "dev.azure.com?owner=<MY_AZURE_PROJECT>&repo=<MY_AZURE_REPOSITORY>&organization=<MY_AZURE_ORGANIZATION>&version=<MY_AZURE_REPOSITORY_BRANCH>"
```

Comment:
We should probably remove this hardcoded host - same as https://github.com/Parfuemerie-Douglas/scaffolder-backend-module-azure-pipelines/issues/19